downlords-faf-client.install4j

```diff
@@ -699,6 +699,30 @@ return console.askOkCancel(message, true);
                 </serializedBean>
                 <condition />
               </action>
+              <action name="Kill left over ice adapters" id="621" customizedId="" beanClass="com.install4j.runtime.beans.actions.control.RunScriptAction" enabled="true" commentSet="false" comment="" actionElevationType="elevated" rollbackBarrier="false" rollbackBarrierExitCode="0" multiExec="false" failureStrategy="1" errorMessage="">
+                <serializedBean>
+                  <java class="java.beans.XMLDecoder">
+                    <object class="com.install4j.runtime.beans.actions.control.RunScriptAction">
+                      <void property="script">
+                        <object class="com.install4j.api.beans.ScriptProperty">
+                          <void property="value">
+                            <string> try{            
+     Runtime rt = Runtime.getRuntime();
+     Process proc = rt.exec("for /f \"tokens=1\" %i in ('jps -m ^| find \"faf-ice-adapter\"') do ( taskkill /F /PID %i )");
```

Comment:
I don't like that, but I don't think there is an alternative.

Reply:
Well I suspect it won't hurt right. Also I need to test that first to